demo/index.html

```diff
@@ -7,6 +7,7 @@
   <script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
   <link rel="import" href="../google-map.html">
   <link rel="import" href="../google-map-marker.html">
+  <link rel="import" href="../google-map-poly.html">
```

Comment:
can we make another demo file for this instead of adding it to the basic maps demo?